README.md

```diff
@@ -283,3 +283,94 @@ Not all required parameters are provided, since deploy specific, see [report-gen
+-   **lblodlg:Functionaris:** this models a Leidinggevende in the local government
+-   **schema:ContactPoint:** to model the contact point of the Functionaris above
+-   **locn:Address:** to model the address of the Functionaris above
+-   **ext:DraftMandataris:** a type to model the draft Mandataris instances for GN and make it clear that they are different from the mandaat:Mandataris instances that are ready to be published
+
+The only predicates that are shared about these concepts are predicates that are defined in the application profiles for [mandatendatabank](https://data.vlaanderen.be/doc/applicatieprofiel/mandatendatabank/), [leidinggevenden](https://lblod.github.io/pages-vendors/#/docs/leidinggevenden) and [besluit-publicatie](https://data.vlaanderen.be/doc/applicatieprofiel/besluit-publicatie/).
+
+## Mandataris as a Versioned Entity
+
+A Mandataris is a Versioned Entity, meaning that when one wants to model a change in state of Mandataris from e.g. 'actief' to 'verhinderd' at time `T`, the following should happen:
+-   the current Mandataris entity keeps its `mandaat:status`, but the `mandaat:einde` changes to `T`. Let's call this original Mandataris entity `O`.
+-   a new Mandataris entity is created with the same content as the first, but without a `mandaat:einde` and with a `mandaat:start` set to `T`. This new Mandataris entity receives the new `mandaat:status` of 'verhinderd', lets call this new Mandataris entity `V`.
+-   if one wants to model that another Mandataris replaces the first, another Mandataris entity `R` is created and the new 'verhinderd' entity `V` receives a `mandaat:isTijdelijkVervangenDoor`  reference to `R`.
+
+This means that except for correcting mistakes and changing the `mandaat:einde`, Mandataris entities should never be modified.
+
+## Expected Peripheral Knowledge
+
+There is a substantial amount of data that is referenced in the entities on the LDES above but that is not expected to change often (or at all) and that isn't the responsibility of the LMB system to maintain. Let's call such data Peripheral Knowledge.
+
+As the LMB LDES instances reference this data, we do expect systems that the LDES data to have this Peripheral Knowledge loaded themselves. 
+
+A suggestion would be that a central system could provide its own LDES feed that other systems can use to update their peripheral knowledge (given how little this data is expected to change, this can even be a manual update). There are multiple options here and this is something that can wait as we will likely only launch centrally first (possibly one separately with heavy, heavy support). In this case, a migration or set of migrations will be good enough. Another path is that we can make an interface (API, not LDES) where they just fetch the latest version. Another path is to use the caching headers of LDES, allowing caching for e.g. half a year. We can also terminate an LDES when we know no more data is going to change.
+
+## LDES setup
```

Comment:
This paragraph goes a bit over my head, but that's probably because I'm not really in the know about ldes in general

Reply:
no. will rewrite